app/templates/admin/view_tags.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable grid container">
+        <div class="sixteen wide tablet twelve wide computer centered column">
+            <a class="ui basic compact button" href="{{ url_for('admin.index') }}">
+                <i class="caret left icon"></i>
+                Back to dashboard
+            </a>
+            <h2 class="ui header">
+                Tags
+            </h2>
+
+          <div class="row">
+            <div class="col-md-4">
+              <select class="custom-select" id="select-role">
+                <option  class="item" value="">All account types</option>
+                {% for r in tag_types %}
+                <option  class="item" value="{{ r.tag_type_name }}">{{ r.tag_type_name }}</option>
+                {% endfor %}
+              </select>
+            </div>
+            <div class="col-md-8">
+              <div class="form-group search item">
+                <input id="search-users" class="form-control" type="text" placeholder="Search tags or tag types…">
+              </div>
+            </div>
+          </div>
+
+            {# Use overflow-x: scroll so that mobile views don't freak out
+             # when the table is too wide #}
+             <button onclick="window.location.href = '{{ url_for('admin.add_new_tag') }}';" 

```

Comment:
can you add some spacing between the above fields and this button?

Reply:
not exactly sure how much spacing you want to do so i did margin-bottom:20px so lmk if that looks good to you.